packages/router/src/grpc/view-protocol-server/ephemeral-address.ts

```diff
@@ -3,13 +3,17 @@ import { servicesCtx } from '../../ctx';
 
 import { getEphemeralByIndex } from '@penumbra-zone/wasm-ts';
 
+// TODO handle req.displayConfirm
```

Comment:
I couldn't find any documentation about this field, and also it's not implemented in Rust

Reply:
Hmm. Going to ask in discord.